scripts/ci/test_extensions.sh

```diff
@@ -23,7 +23,8 @@ exit_code=0
 # Disable arcappliance arcdata connectedk8s: https://github.com/Azure/azure-cli/pull/20436
 # Disable k8s-extension temporarily: https://github.com/Azure/azure-cli-extensions/pull/6702
 # Disable alias temporarily: https://github.com/Azure/azure-cli/pull/27717
-ignore_list='azure-cli-ml fzf arcappliance arcdata connectedk8s k8s-extension alias'
+# Disable db-up temporarily: https://github.com/Azure/azure-cli/pull/29887
+ignore_list='azure-cli-ml fzf arcappliance arcdata connectedk8s k8s-extension alias db-up'
```

Comment:
`db-up`'s `pymssql==2.2.7` does not provide wheel for 3.12. Updating the version can fix this. It's currently prepare for beta release, so I can't release a new version now.

https://github.com/Azure/azure-cli-extensions/blob/c55f55aaf70fd5df20b5a789294bb909aab4d1ce/src/db-up/setup.py#L33